### PR TITLE
Create Disk Detective Archived Project

### LINKED ISSUE #11: Create 'interim' page for DiskDetective.org
For April 30th Ouroborous shutdown date, create 'interim' page for DiskDetective.org, following the template/approach as for https://whaling.oldweather.org.

(It's an 'interim' page because Kuchner and co are in process of rebuilding on PFE, but won't be ready for beta till summer). 

I expect to have content (a few sentences of text and an image) from Kuchner by Friday noon. 

Note: https://talk.diskdetective.org/ should persist as is.

### PR DESCRIPTION
**Pull request by wgranger**
This PR creates the Disk Detective archived page. A few notes here:

- A JS array was created to hold all user names for the static "authors" page 
- Section ids are set to /index or /authors to account for the original authors path we want to continue supporting

Closes #11 

Screen Shot 2019-05-01 at 10 55 06 AM
Screen Shot 2019-05-01 at 10 55 18 AM

### CODE REVIEW

**Review by srallen — APPROVED** [review]

File: DiskDetective/index.html [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        </div>
+      </div>
+    </section>
+
+    <section id="/index">
+      <div class="jumbotron">
+        <div class="container container-flex">
+          <img src="img/disk.png" class="avatar cover"/>
+          <div>
+            <h1>Disk Detective</h1>
```
Comment (eatyourgreens) [inline DiskDetective/index.html:57]: This section should be the first section on the page. At the moment, you have h2 headings ahead of h1.

Comment (srallen) [reply DiskDetective/index.html:57]: This is working like a single page app, so the other two sections are display none while this is rendered.

Comment (eatyourgreens) [reply DiskDetective/index.html:57]: That doesn't matter to a HTML parser, though. If you want this page to pass accessibility checks, and have decent SEO, H1 should be the first heading on the page.

Comment (srallen) [reply DiskDetective/index.html:57]: Ok, good to know 👍 . @wgranger this update should be made to any of the sites that have the extra static pages you've implemented as a SPA.

Comment (wgranger) [reply DiskDetective/index.html:57]: Gotcha. Yeah, the h1 was put at the end so the css would display each section directly when targeted, (with #index appearing when no target present). I bet I can find another solution.

**Pull request merged**